docs/iris/src/developers_guide/dask_interface.rst

```diff
@@ -1,23 +1,35 @@
 Iris Dask Interface
 *******************
 
-Iris uses dask (http://dask.pydata.org) to manage lazy data interfaces and processing graphs.  The key principles which define this interface are:
-
-* A call to `cube.data` will always load all of the data.
-  * Once this has happened:
-    * `cube.data` is a mutable numpy masked array or ndarray;
-    * `cube._numpy_array` is a private numpy masked array, accessible via `cube.data`, which may strip off the mask and return a reference to the bare ndarray.
-* `cube.data` may be used to set the data, this accepts:
-  * a numpy array (including masked array), which is assigned to `cube._numpy_array`;
-  * a dask array, which is assigned to `cube._dask_array` an `cube._numpy_array` is set to None.
-* `cube._dask_array` may be None, otherwise it is expected to be a dask graph:
```

Comment:
This is **not** expected to be a dask graph! A dask graph is a dictionary describing processing steps for a scheduler to execute. A dictionary would not make a good data payload for an Iris cube.

For an example of a dask graph see http://dask.pydata.org/en/latest/graphs.html#example.